rules-reviewed/openshift/local-storage.windup.xml

```diff
@@ -54,7 +49,8 @@ It is critical to do an audit and find where your application uses local files,
             </when>
             <perform>
                 <hint title="Used class java.net.URL/URI with local path" effort="1" category-id="optional">
-                    <message>OpenShift environment **should not** use local storage for persistent data, but instead of that use a **database** or **cloud native network file services**.</message>
+                    <message>Local files in containers are not permanent storage and can disappear when an instance of the application terminates. Avoid accessing local filesystems from applications running in OpenShift environments, and consider using a database or cloud-based storage system instead.
```

Comment:
This message is the same as above message for local-storage-00001. We should have unique message for every hint, otherwise it should be merged into one.